actor/builtin/miner/miner.go

```diff
@@ -714,3 +723,75 @@ func (ma *Actor) GetProvingPeriodStart(ctx exec.VMContext) (*types.BlockHeight,
 
 	return state.ProvingPeriodStart, 0, nil
 }
+
+func (ma *Actor) currentProvingPeriodPoStChallengeSeed(ctx exec.VMContext) (proofs.PoStChallengeSeed, error) {
+	chunk, err := ctx.ReadStorage()
```

Comment:
Good call. I'll modify this method:

1) make it a function instead of a method on `Actor`
2) make it take a `State` and a `exec.VMContext` instead of just an `exec.VMContext`